backend/src/payment/index.ts

```diff
@@ -18,6 +19,8 @@ import express = require('express');
  * Provider implementations can be found from folder ./providers
  */
 
+ const PAYMENT_PREFIX = 'NAPPIKAUPPA2_';
```

Comment:
Is this same or different than we were using previously? If the name is visible to customer then NAPPIKAUPPA2 might not be the best, but if not then it's good.

Reply:
Changed to LIPPUKAUPPA_ as it is visible at least in checkout